### PR TITLE
Data/classes

### PR DESCRIPTION
**Pull request by jphorner**
**Additions**

- Created WIP class files with essential properties
- Built out base HTML and SCSS
- Set up starting API calls
- Implemented a few tests on most important classes
  - More will be added later on

**Issues**

- Dealt with a merge issue for two days, hence the late PR. Sorry for the delay!

**Other**

- _Please ignore scripts.js_
  - I was guided through this code as a demonstration by a classmate. It does not apply to my repo and I will not use it in the end; I simply wanted to get the code viewable in my console before I continued. Will delete and rewrite once I start writing scripts.

### CODE REVIEW

File: src/classes/Booking.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Room from './Room.js';
+import Customer from './Customer.js'
```
Comment (PaulTimothyChambers) [inline src/classes/Booking.js:2]: I'm curious why you're importing your room and customer classes in here...are you planning on instantiating them from within the booking class?

Comment (jphorner) [reply src/classes/Booking.js:2]: I was initially - I tried to set the classes up in anticipation of the functionality I might need. Now that I'm farther into the project I see that it wasn't necessary. I'll get rid of those.

File: src/css/main.css.map [hunk 2]
```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["base.scss","_variables.scss"],"names":[],"mappings":"AAAA;AAOA;EACE;EACA;EACA;EACA;;;AAGF;EACE;EACA;EACA;EACA;EACA,YClBmB;EDmBnB,aCVa;;;ADqBf;EACE;EACA;EACA;EACA;EACA;EACA;EACA;;AAEA;EACE;EACA;EACA;EACA;EACA;;AAGF;EACE;EACA;EACA;EACA;EACA;EACA;EACA;;AAGF;EACE;EACA;EACA,OCxDU;EDyDV,aCnDa;EDoDb;;;AAKF;EACE;EACA;;;AAIJ;EACE;EAEA;;;AAGF;EACE;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;EACA;;AAEA;EACE;EACA;;AAGF;EACE;EACA;;AAGF;EACE;EACA;;;AAIJ;EACE;EACA;EACA;EACA;;;AAGF;EACE","file":"main.css"}
```
Comment (PaulTimothyChambers) [inline src/css/main.css.map:1]: What is this map for? I've never seen anything like it, or if I have I don't remember :p

**Review by PaulTimothyChambers — APPROVED** [review]

File: src/css/main.css.map (hunk 2 above)
Comment (jphorner) [reply src/css/main.css.map:1]: Whoops! I shouldn't have pushed that up. That's the file used to watch the changes I made to the SCSS and map them to CSS. ^^;

**Pull request merged**